integrations/external_ip.go

```diff
@@ -0,0 +1,22 @@
+package integrations
+
+import (
+	"fmt"
+	"net"
+)
+
+const NoExtIPApi = "no external ip address API available"
+
+// GetExternalIP
+// function to get the external ip from any avaliable external ip integration
+func GetExternalIP() (err error, IP net.IP) {
```

Comment:
Codacy found an issue: [error should be the last type when returning multiple items](https://app.codacy.com/gh/go-go-dojo/friendly-potato/pullRequest?prid=6165294)